fixtures/js/eslint/package.json

```diff
@@ -13,12 +13,11 @@
   "devDependencies": {
     "@babel/core": "^7.7.2",
     "babel-eslint": "^10.0.3",
-    "eslint": "^6.6.0",
-    "eslint-config-prettier": "^6.7.0",
-    "eslint-plugin-prettier": "^3.1.1",
-    "husky": "^3.1.0",
-    "lint-staged": "^9.4.3",
-    "prettier": "^1.19.1"
+    "eslint": "^8.21.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "husky": "^8.0.1",
```

Comment:
@sandeep-vedam can you test what the impact would be of this major version update of Husky from v3 to v8? In version 4 Husky seems to have changed their config quite a bit (see also issue #198).

Same goes in general for all these major version bumps.

Reply:
I tested out eslint pretty well but unfortunately I missed husky. After Husky v4 they changed how you specify and install the git hooks. I've updated the PR to fix it

Reply:
Cool thanks for these updates

@sandeep-vedam can you run a quick sanity check with the new husky config please? Just to confirm it all continues to work well (on different operating systems)?

Reply:
Hey @michielvandergeest I have tested it out on both Ubuntu and Windows. It works perfectly on Ubuntu.

But on Windows we are getting an error 'error: cannot spawn .husky/pre-commit: No such file or directory' when we commit.

@frank-weindel ,  please have a look at this. 
Fix:  
1. Based on the os we need to copy './husky/pre-commit' to the project that is being created. For windows we need to add a line '#!/bin/bash' in pre-commit file.